docs/cloud/pouta/heat-orchestration.md

```diff
@@ -64,45 +71,167 @@ stacks:
     | 98077bd5-9d69-47c3-98db-b0e19a60b1fa | my-heat-stack | CREATE_COMPLETE | 2016-06-08T07:34:46Z | None         |
     +--------------------------------------+---------------+-----------------+----------------------+--------------+
 
-### Heat guidelines and command references
+### Explanations
+A template is composed of two major sections: 
+- The version used (`heat_template_version`)
+- The resource(s) (`resources`)
 
-For more information, visit the OpenStack
-[Heat wiki]. For a full reference
-to the OpenStack command line client, see the [command line
-reference], and for the Heat command
-line client, see the [Heat CLI reference].
+There are also optional sections like:
+- The description (`description`)
+- The parameter group(s) (`parameter_groups`)
+- The parameter(s) (`parameters`)
+- The output(s) (`outputs`)
+- The condition(s) (`conditions`)
 
-To try out the example provided here, modify at least the "key\_name"
-in the template code:
+Here is a detail of each sections:  
 
-    heat_template_version: 2015-10-15
+`heat_template_version`  
+&nbsp;&nbsp;&nbsp;&nbsp; Indicates that the YAML document is a HOT template of the specified version.
```

Comment:
What does `HOT` mean here?

Reply:
HOT stands for Heat Orchestration Template. Fixed in the doc